test/arithmetic/divider_test.dart

```diff
@@ -399,6 +410,84 @@ class MultiCycleDividerBasicSequence extends Sequence {
   }
 }
 
+class MultiCycleDividerEvilSequence extends Sequence {
+  late final int numBits;
+
+  MultiCycleDividerEvilSequence(
+      {required this.numBits, String name = 'MultiCycleDividerEvilSequence'})
+      : super(name);
+
+  @override
```

Comment:
Great test for the corner cases! Are there other corners that could use a similar sequence (zero, one, etc.)

Reply:
Yeah any suggestions? Divide by zero is covered in the basic sequence. Maybe 0 divided by some non-zero number? I tried to cover all of the corners I could think of with this evil sequence already.